.github/workflows/pull-request-management.yml

```diff
@@ -103,7 +103,7 @@ jobs:
           docker-compose -f docker-compose.yml ps
       - name: 'Test connectivity to mkdoc server'
         run: |
-          until docker exec webdoc_avd curl -s -I http://localhost:8000/ ; do sleep 2; done
+          bash -c 'while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' 127.0.0.1:8000)" != "200" ]]; do sleep 5; done'
```

Comment:
Should we add a timeout on this to avoid the default 12-hour timeout for the job?

Reply:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepstimeout-minutes

Reply:
ill get a timeout added to that step. thank you

Reply:
considering the length of our ci, thoughts on doing something like 40 mins?

Reply:
disregard for that step ill do something much shorter

Reply:
set for 10 minutes, pipeline usually completes in 5 mins